scripts/deploySidechain.ts

```diff
@@ -1138,3 +1140,51 @@ export async function deploySidechainPhase3(
 
     return { stashRewardDistro, childGaugeVoteRewards };
 }
+
+export async function deploySidechainAuraLocker(
+    hre: HardhatRuntimeEnvironment,
+    signer: Signer,
+    multisigs: SidechainMultisigConfig,
+    naming: SidechainNaming,
+    extConfig: ExtSidechainConfig,
+    sidechain: SidechainPhase1Deployed,
+    debug = false,
+    waitForBlocks = 0,
+    salt: string = SALT,
+): Promise<{ cvxLocker: AuraLocker }> {
+    const create2Options = { amount: 0, salt, callbacks: [] };
+    const deployOptions = {
+        overrides: {},
+        create2Options,
+        debug,
+        waitForBlocks,
+    };
+
+    const deployOptionsWithCallbacks = (callbacks: string[]) => ({
+        ...deployOptions,
+        create2Options: {
+            ...create2Options,
+            callbacks: [...callbacks],
+        },
+    });
+
+    const create2Factory = Create2Factory__factory.connect(extConfig.create2Factory, signer);
+
+    const cvxLockerTransferOwnership = AuraLocker__factory.createInterface().encodeFunctionData("transferOwnership", [
+        multisigs.daoMultisig,
+    ]);
+    const cvxLocker = await deployContractWithCreate2<AuraLocker, AuraLocker__factory>(
+        hre,
+        create2Factory,
+        new AuraLocker__factory(signer),
+        "AuraLocker",
+        [naming.vlCvxName, naming.vlCvxSymbol, sidechain.auraOFT.address, ZERO_ADDRESS, ZERO_ADDRESS],
+        deployOptionsWithCallbacks([cvxLockerTransferOwnership]),
+    );
+    if ((await cvxLocker.cvxCrv()) !== ZERO_ADDRESS) {
```

Comment:
This seems uneeded, cvxCRV is always going to be ZERO because you are passing in ZERO into the constructor. Can remove this.